lib/src/view/check_connected_device_online_screen.dart

```diff
@@ -13,6 +13,7 @@ class _CheckConnectedDeviceOnlineScreenState extends State<CheckConnectedDeviceO
   @override
   void initState() {
     super.initState();
+    widget.viewModel.reconnect(); // reconnect to device if it was disconnected
```

Comment:
need this to for tethering (won't impact normal ble provisioning, reconnect only happens if you're disconnected in the repo) - so we can read any errors from agent if it fails to come online